data_finder.py

```diff
@@ -0,0 +1,51 @@
+import twitter
+import pickle
+#who do you want to mimic
+def data_finder(user_handle):
+    while True:
+        #data finder prompst the user to input login material
+        print ('consumer_key:')
+        c_k = input()
+        print ('consumer_secret:')
+        c_s = input()
+        print ('access_token_key:')
+        a_t_k = input()
+        print ('access_token_secret:')
+        a_t_s = input()
+        #data fider uses these to login
+
+        api = twitter.Api(consumer_key=c_k,
+                          consumer_secret=c_s,
+                          access_token_key=a_t_k,
+                          access_token_secret=a_t_s)
+
+        # test to see if this brings an error if it doesnt break! if it does,
+        #try autenticating again
+        try:
+            api.VerifyCredentials()
+            break
+        except twitter.error.TwitterError as e:
+            print(e)
+            data_finder(user_handle)
+
+    # how do i make a list of lists of words from each tweet?
+    # How do i set up a login error that brings people back to the login option and works for more people than just me
+    # How do i save that list so i can open it in another progam
```

Comment:
If you want to use the list in another program, one way to do it is by pickling the list.